tsconfig.json

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Google LLC
+ * Copyright 2019 Google LLC
```

Comment:
why this change? seems weird.

Reply:
came over with one of the files moves, I can put it back